core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -119,7 +119,7 @@
 	private int correlationIdMaxLength = -1;
 
 	public enum AcknowledgeMode implements DocumentedEnum {
-		@EnumLabel("") NOT_SET(0),
+		@EnumLabel("none") NOT_SET(0),
 		@EnumLabel("auto") AUTO_ACKNOWLEDGE(Session.AUTO_ACKNOWLEDGE),
 		@EnumLabel("client") CLIENT_ACKNOWLEDGE(Session.CLIENT_ACKNOWLEDGE),
 		@EnumLabel("dups") DUPS_OK_ACKNOWLEDGE(Session.DUPS_OK_ACKNOWLEDGE);
```

Comment:
Voor de volledigheid zou hier nog javadoc bij kunnen/moeten met de beschrijving die je op regel 828 weg hebt gehaald. Ook is het handig als er een meer geschrijvende tekst bij komt zoals:

`/** auto or auto_acknowledge: Specifies that the session is to automatically acknowledge consumer receipt of messages when message processing is complete. */ `

`/** dups or dups_ok_acknowledge: Specifies that the session is to "lazily" acknowledge the delivery of messages to the consumer. "Lazy" means that the consumer can delay the acknowledgment of messages to the server until a convenient time; meanwhile the server might redeliver messages. This mode reduces the session overhead. If JMS fails, the consumer may receive duplicate messages. */ `

`/** client or client_acknowledge: Specifies that the consumer is to acknowledge all messages delivered in this session. */ `

Reply:
Er moet zowel javaDoc bij de class, die dan beschrijft wat AcknowledgeMode is, en bij elke waarde, die die specifieke waarde beschrijft